README.md

```diff
@@ -7,20 +7,6 @@ With this project, we want to provide an example and starting point for C++ proj
 The project has initially been forked/copied from [Jason Turner's cpp_starter_project](https://github.com/lefticus/cpp_starter_project).
 
 
-<hr>
```

Comment:
This change does not appear to be part of the GitHub Action change. Also, removing the note that this was the original README makes it appear as though it is our README while some sections still have references to the GUI libraries and links to lefticus' project. Shall we leave the README changes to a separate PR where we rework everything thoroughly?

Reply:
I will remove this file from commit and then this should be solved on a different PR